packages/saber/lib/utils/assetsAttribute.js

```diff
@@ -0,0 +1,44 @@
+// @ts-check
+
+/**
+ * Check if it's external resource
+ * @param {string} str
+ */
+const isExternal = str => /^https?:\/\//.test(str) || /^\//.test(str)
+
+const MARK = '@@!!SABER_ASSET_MARK_e5968b9a!!@@'
```

Comment:
Is `e5968b9a` there just to make it more random?

Reply:
Yep 😂

Reply:
So there's still a chance I won't break anything even if I name my assets `@@!!SABER_ASSET_MAR_!!@@cute-doggo.jpg`, that's awesome :laughing: